src/playlist-selectors.js

```diff
@@ -352,3 +353,27 @@ export const minRebufferMaxBandwidthSelector = function(settings) {
 
   return rebufferingEstimates[0] || null;
 };
+
+/**
+ * Chooses the appropriate media playlist, which in this case is the lowest bitrate
+ * one with video.  If no renditions with video exist, return the lowest audio rendition.
+ *
+ * Expects to be called within the context of an instance of HlsHandler
+ *
+ * @return {Object|null}
+ *         {Object} return.playlist
+ *         The lowest bitrate playlist that contains a video codec.  If no such rendition
+ *         exists pick the lowest audio rendition.
+ */
+export const lowestBitrateCompatibleVariantSelector = function() {
+  const playlists = this.playlists.master.playlists.slice();
+
+  // Sort ascending by bitrate
+  stableSort(playlists,
```

Comment:
Based on the findings from the 404 unit test failing (discussion in other comment thread), this selector should probably filter by `isEnabled` similar to the `simpleSelector` since this may try and re-select a playlist that just 404'd